README.md

```diff
@@ -151,16 +151,20 @@ The `auth` option can be
    - GitHub App bearer tokens
    - GitHub App installation tokens
 
-2. As object with the properties `username`, `password`, `on2fa`.
+2. As object with the properties `username`, `password`, `on2fa`, `note`, `scopes`.
 
    `on2fa` is an asynchronous function that must resolve with two-factor
    authentication code sent to the user.
 
+   `note` and `scopes` are used in the [creation of a token](https://octokit.github.io/rest.js/#api-OauthAuthorizations-createAuthorization).
```

Comment:
I think I would remove this paragraph and instead put a paragraph on top of the section explaining that Basic Authentication is used to create a token, which in turn is used for subsequent requests.